boards/arm/nrf7002dk_nrf5340/nrf5340_cpuapp_common.dts

```diff
@@ -150,6 +151,10 @@ arduino_i2c: &i2c1 {
 	pinctrl-0 = <&uart0_default>;
 	pinctrl-1 = <&uart0_sleep>;
 	pinctrl-names = "default", "sleep";
+	nrf_wlan0: nrf_wlan0 {
```

Comment:
why is this under UART? Shouldn't this be a separate node? And the convention seems to be using `nordic`?

Reply:
Moved to the root node.. Vendor name convention is limited to compatible property [(ref).](https://docs.zephyrproject.org/2.7.5/guides/dts/intro.html#important-properties) .  `nrf_wlan0` should be fine for node label. But let me know if you want me to change this.

Reply:
Yeah, using both `nrf` and `nordic` seems a bit weird, but I don't have strong opinions.

Reply:
Changed to `nordic` instead of using `nrf`